fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/JDBCQueryBuilder.java

```diff
@@ -648,8 +648,13 @@ protected SqlQueryData processChainedReferenceParm(QueryParameter queryParm) thr
             } else {
                 // This logic processes the LAST parameter in the chain.
                 // Build this piece: CPx.PARAMETER_NAME_ID = x AND CPx.STR_VALUE = ?
+                if (chainedParmVar == null) {
+                    chainedParmVar            = CP + 1;
+                }
                 Class<?> chainedResourceType = ModelSupport.getResourceType(resourceTypeName);
                 SqlQueryData sqlQueryData = buildQueryParm(chainedResourceType, currentParm, chainedParmVar);
+
+                // log.info("chained sqlQueryData[" + chainedParmVar + "] = " + sqlQueryData.getQueryString());
```

Comment:
I'm ok with it left there, probably should wrap in is loggable fine and log.fine

Reply:
wrapped and changed to fine